package.json

```diff
@@ -45,6 +45,7 @@
     "is-ci": "^3.0.0",
     "isbinaryfile": "^4.0.6",
     "jest": "^26.6.3",
+    "jest-utils-lwc-internals": "link:./scripts/jest/utils",
```

Comment:
I've never seen npm `link:` dependencies before. Why not a separate package in the monorepo, or just a relative path to the script when importing?

Reply:
I sticked to the pattern we used for the eslint plugin. We can move both the eslint plugin and the internal test library to a standard package in the monorepo in another PR.